src/components/question/question.js

```diff
@@ -0,0 +1,21 @@
+import './question.css'
+import arrow from '../../images/icon-arrow-down.svg'
+
+const Question = ({question,answer}) => {
+    const handleclick=(e)=>{
+        e.currentTarget.classList.toggle("visible");
```

Comment:
It's not really good way for handling such case in react. You can use state for identify that element should be visible. Take a look https://daveceddia.com/usestate-hook-examples/